### ISSUE TITLE
Reduce size of ASGS git repo

### ISSUE THREAD

**Issue by jasonfleming** [StormSurgeLive/asgs#925]
The ASGS git repo is over 100MB for no good reason and this causes cloning to be slow over slow internet connections. I tried the following to have a look at the worst offenders: 
```
git rev-list --objects --all | git cat-file --batch-check='%(objecttype) %(objectname) %(objectsize) %(rest)' | awk '/^blob/ {print substr($0,6)}' | sort --numeric-sort --key=2 | cut --complement --characters=13-40 | numfmt --field=2 --to=iec-i --suffix=B --padding=7 --round=nearest | tail
```
 and saw the following:
```
b9f4c77ddbf7  6.6MiB doc/ASGSOperatorsGuide.pdf
584be0ba21cd  6.8MiB output/nc_inundation_v6b_msl-inundation-masks.nc
30d1bf991c2a  8.8MiB input/meshes/hsofs/archive/60cmNGOffset.dat
08399e705063  9.8MiB input/meshes/nc_v9.99_w_rivers/oi_surface_nc_inundation_v9.99_rivers_0.1524.dat
9200db727840   10MiB input/meshes/nc_v9.99_w_rivers/unit_offset_nc_inundation_v9.99_rivers.dat
c89814ed1b39   24MiB input/meshes/LA_v12g-WithUpperAtch/60cmNGOffset.dat
a0eefa749ec6   26MiB tides/ec2001.grd
94fd9096d155   29MiB input/meshes/hsofs/unit_offset_hsofs.dat
a174c62ee3a1   31MiB input/v6brivers.13
16d10f2af46d   34MiB input/v6brivers.14
```
If we could even just remove the last 3, it would be a significant improvement. Not sure how disruptive that would be.

**Comment by wwlwpd** [comment]
Nice. This would be very nice to do.

**Comment by wwlwpd** [comment]
This is going to require use deleting a bunch of history; basically archive the "old" repo, create a hard fork at a point where to carry no history prior. I think we should do still do it. Have far back do you want to fork it? It'd need to be far enough back to miss the heavy files we're carrying history, but allow for anyone using an older version to catch up.

**Comment by jasonfleming** [comment]
Is there another way? I thought we could delete the commits that put these large files in the repo, then re-SHA the later commits. So the history would be the same but all the SHAs after the large file removal would be updated. Kind of like a `rebase` _en masse_. Is that possible?

**Comment by wwlwpd** [comment]
Yes, I looked into it some time ago, but it is a bear to run and requires some manual intervention. It really needs to be scripted. I will look more into that. This problem is not present on releases (.tgz or .zip), which exports the specified branch or tag.

e.g., with the latest RC we're sitting at 63 MB:

```
[ASGS (outside repo)] stampede3@default-asgs-prod> du -h 2025.2.rc-4.tar.gz
63M     2025.2.rc-4.tar.gz
```

**Comment by jasonfleming** [comment]
Yes this is a can of worms. Closing for now, we can reopen it if we want to revisit.
